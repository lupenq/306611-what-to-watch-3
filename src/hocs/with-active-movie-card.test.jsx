import withActiveMovieCard from '../hocs/with-active-movie-card';

const IMG_URL = `https://api.adorable.io/avatars/128`;

const films = {
  name: `Интерстеллар`,
  picture: `${IMG_URL}`,
  id: 1,
  play: true,
  preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
};

const MockComponent = () => <div></div>;
const MockComponentWrapped = withActiveMovieCard(MockComponent);

it(`withActiveMovieCard is rendered correctly`, () => {
  const tree = renderer
    .create(
        <MockComponentWrapped
          name={films.name}
          picture={films.picture}
          key={films.id}
          id={films.id}
          preview={films.preview}
          onMovieCardTitleClick={() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
