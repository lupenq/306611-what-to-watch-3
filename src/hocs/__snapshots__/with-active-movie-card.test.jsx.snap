// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveMovieCard is rendered correctly 1`] = `<div />`;
